good-enough-practices-for-scientific-computing.tex

```diff
@@ -150,9 +150,12 @@ \section*{Author Summary}
 
 Computers are now essential in all branches of science, but most
 researchers are never taught the equivalent of basic lab skills for
-research computing. As a result, their analyses can take much longer
-than necessary, and they may not be able to reproduce their work or
-make it reproducible by others. 
+research computing. As a result, data can get lost, analyses can take 
+much longer than necessary and researchers are limited in how effectively
+and they can work with software and data. Computing workflows need to
+follow the same practices as with lab projects and notebooks, with organized data,
+documented steps and the project structured for reproducibility, but 
```

Comment:
'and a project structure for reproducibility' or 'and the project structured for reproducibility' would both work, I think. 

Sentences aren supposed to start with 'but', right? I would support 'reproducibility. However, researchers ...'